.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+# **Problem:**
+
+- Describe the specific technical problem that this PR solves.
+- Include  details such as:
+    - How you reproduced the problem
+    - What code is affected
+    - Any relevant background information
+
+# **Solution:**
+
+- Describe the specific technical solution that this PR provides.
+- Include details such as:
+    - How you debugged the problem
+    - Why you chose this solution
+    - Does the solution make any impact beyond the immediate problem
+    - Was there any necessarytechnical debt incurred
+    - Any important design decisions/links todesign documentation
+
+# **Testing:**
+
+- Describe any testing that was performed to validate the solution.
+- Include details such as:
+    - How you tested the solution
+    - Any relevant test results
```

Comment:
I wonder if we should include a warning about sharing test results if they contain sensitive or info that should be more private that could be included in JIRA ticket instead. Just thinking out loud here but for Genie, we have staging test runs that run on actual samples of datasets and we upload those comparison reports to private synapse folders and link to them in JIRA

Reply:
+1 to this

Reply:
This is a great point, thanks for your perspective @rxu17 ! I added a warning like this in the latest [commit](https://github.com/Sage-Bionetworks-Workflows/py-dcqc/pull/58/commits/f6ee0714da200d059a3ae2eeaff266cb19353037).